packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1606,8 +1606,6 @@ export class ContainerRuntime
 				!readonly || !this.connected,
 				0x125 /* "Unsafe to transition to read-only state!" */,
 			);
-
-			this.replayPendingStates();
```

Comment:
Is the comment above still relevant? cc @vladsud

Reply:
As of "2.0.0-internal.1.2.0", we set the connection state in Container on the DeltaManager's "readonly" event.
https://github.com/microsoft/FluidFramework/pull/11655

Maybe this call should be kept for compat with lts version? This place would also need similar Outbox flushing.